src/main/resources/META-INF/native-image/org.jboss.threads/jboss-threads/native-image.properties

```diff
@@ -0,0 +1 @@
+Args = --initialize-at-build-time=org.jboss.threads.EnhancedQueueExecutor$RuntimeFields
```

Comment:
Shouldn't it initialize at Run time instead? 
```suggestion
Args = --initialize-at-run-time=org.jboss.threads.EnhancedQueueExecutor$RuntimeFields
```

Reply:
Thank you for catching this @gastaldi!!!!